src/rviz/default_plugin/map_display.cpp

```diff
@@ -212,6 +213,22 @@ unsigned char* makeCostmapPalette()
   return palette;
 }
 
+unsigned char* makeRawPalette()
+{
+  unsigned char* palette = new unsigned char[256*4];
```

Comment:
This is never `delete`'ed. I see that the other `make*Palette()` functions do the same... I guess this is fine as is, but really they should all be updated to use static arrays or something like that.